packages/client/components/ActivityLibrary/ActivityCard.tsx

```diff
@@ -1,68 +1,96 @@
+import strategyBackgroundSrc from '../../../../static/images/illustrations/strategy-background.png'
+import premortemBackgroundSrc from '../../../../static/images/illustrations/premortem-background.png'
+import postmortemBackgroundSrc from '../../../../static/images/illustrations/postmortem-background.png'
+import {upperFirst} from '../../utils/upperFirst'
+import {MeetingTypeEnum} from '../../__generated__/NewMeetingQuery.graphql'
+import {CategoryID, CATEGORY_TEXT_COLORS, MEETING_TYPE_TO_CATEGORY} from './Categories'
 
 export interface CardTheme {
   primary: string
   secondary: string
 }
 
-export const ActivityCardImage = (
-  props: PropsWithChildren<React.ImgHTMLAttributes<HTMLImageElement>>
-) => {
-  const {className, src} = props
+const backgroundImgMap = {
+  retrospective: retroBackgroundSrc,
+  standup: standupBackgroundSrc,
+  feedback: feedbackBackgroundSrc,
+  estimation: estimationBackgroundSrc,
+  strategy: strategyBackgroundSrc,
+  premortem: premortemBackgroundSrc,
+  postmortem: postmortemBackgroundSrc
+} as const
 
-  return (
-    <div
-      className={clsx(
-        'my-1 flex flex-1 items-center justify-center overflow-hidden px-4',
-        className
-      )}
-    >
-      <img className={'h-full w-full object-contain'} src={src} />
-    </div>
-  )
+type ActivityCardImageProps = {
+  className?: string
+  src: string
+  category: CategoryID
 }
 
-const ActivityCardTitle = (props: ComponentPropsWithoutRef<'div'>) => {
-  const {children, className, ...rest} = props
+export const ActivityCardImage = (props: PropsWithChildren<ActivityCardImageProps>) => {
+  const {className, src, category} = props
+  const backgroundSrc = backgroundImgMap[category] ?? retroBackgroundSrc
 
   return (
     <div
       className={clsx(
-        'px-2 py-1 text-sm font-semibold leading-5 text-slate-800 sm:text-base',
+        'relative flex h-full w-full items-center justify-center overflow-hidden',
         className
       )}
-      {...rest}
     >
-      {children}
+      <img
+        className='absolute z-0 h-full w-full object-cover'
+        src={backgroundSrc}
+        alt='Background'
+      />
+      <img
+        className='absolute top-0 left-0 z-10 h-full w-full object-contain p-10'
+        src={src}
+        alt='Card Illustration'
+      />
     </div>
   )
 }
 
 export interface ActivityCardProps {
   className?: string
   theme: CardTheme
-  titleAs?: React.ElementType
   title?: string
   badge?: React.ReactNode
   children?: React.ReactNode
+  type?: MeetingTypeEnum
 }
 
 export const ActivityCard = (props: ActivityCardProps) => {
-  const {className, theme, title, titleAs, badge, children} = props
-  const Title = titleAs ?? ActivityCardTitle
+  const {className, theme, title, children, type, badge} = props
+  const category = type && MEETING_TYPE_TO_CATEGORY[type]
+  const color = category && CATEGORY_TEXT_COLORS[category].primary
```

Comment:
-1 This is confusing, we're passing in a `theme: CardTheme` which contains the colours, but then we're also deciding some colour here. We should make it consistent. Maybe this should go into the `theme` as well.

Reply:
Sounds good; I've removed `bg-` from the category themes, so we just have the colours. That way, we can add `bg-` or `text-` wherever they're imported